src/hooks/useENS.ts

```diff
@@ -0,0 +1,31 @@
+import { ethers } from "ethers";
+import { useEffect, useState } from "react";
+import { useWeb3Context } from "./web3Context";
+
+const useENS = (address: string) => {
+  const { provider } = useWeb3Context();
+  const [ensName, setENSName] = useState<string | null>(null);
+  const [ensAvatar, setENSAvatar] = useState<string | null>(null);
+  const [loading, setLoading] = useState(false);
```

Comment:
@sasicodes I can see `ensAvatar` and `loading` are not yet used, can we remove them or add them in UI?

/cc @appleseed-iii

Reply:
we also have directions in our README for how to resolve merge conflicts in the translations submodule... can you do that? Then let's let this rip.